docs/vfio-user.rst

```diff
@@ -702,16 +704,23 @@ following format:
 * *size* is the size for the bitmap, in bytes, excluding the VFIO bitmap header.
 * *data* This field is unused in vfio-user.
 
-The VFIO bitmap structure is defined in ``<linux/vfio.h>``
-(``struct vfio_bitmap``).
 
-Each ``struct vfio_bitmap`` entry is followed by the region's bitmap. Each bit
-in the bitmap represents one page of size ``struct vfio_bitmap.pgsize``.
+.. Note::
+   The size of the bitmap can be larger than necessary. The server must still
+   respond with the same bitmap size, zeroing the unused part.
+
+Response
```

Comment:
nit, we use "Reply" elsewhere (and I'm doing so in my rewrite)

Reply:
ok